app/screens/MonthlyBudget/MonthlyBudgetScreen.tsx

```diff
@@ -19,23 +20,33 @@ import navigationOptions from "./MonthlyBudgetScreen.navigationOptions";
 const MIN_MONTHLY_CARBON_BUDGET = 10;
 const MAX_MONTHLY_CARBON_BUDGET = 1000;
 
-const translationMontlyBudgetCountries = [
-  "MONTHLY_BUDGET_SCREEN_LUXEMBOURG",
-  "MONTHLY_BUDGET_SCREEN_UNITED_STATES",
-  "MONTHLY_BUDGET_SCREEN_JAPAN",
-  "MONTHLY_BUDGET_SCREEN_SWEDEN",
-  "MONTHLY_BUDGET_SCREEN_FRANCE",
-  "MONTHLY_BUDGET_SCREEN_CHINA",
-  "MONTHLY_BUDGET_SCREEN_BRAZIL",
-  "MONTHLY_BUDGET_SCREEN_INDIA",
-  "MONTHLY_BUDGET_SCREEN_ETHIOPIA",
+const translationMonthlyBudgetCountries = [
+  ["MONTHLY_BUDGET_SCREEN_LUXEMBOURG", 3500],
+  ["MONTHLY_BUDGET_SCREEN_UNITED_STATES", 1500],
+  ["MONTHLY_BUDGET_SCREEN_JAPAN", 900],
+  ["MONTHLY_BUDGET_SCREEN_SWEDEN", 595],
+  ["MONTHLY_BUDGET_SCREEN_FRANCE", 575],
+  ["MONTHLY_BUDGET_SCREEN_CHINA", 522],
+  ["MONTHLY_BUDGET_SCREEN_BRAZIL", 208],
+  ["MONTHLY_BUDGET_SCREEN_INDIA", 139],
+  ["MONTHLY_BUDGET_SCREEN_ETHIOPIA", 8.3],
 ];
 
-const CountryExample = (translation, index) => (
-  <Text.Secondary center key={index} style={styles.worldExampleItem}>
-    {t(translation)}
-  </Text.Secondary>
-);
+const CountryExample = (translation, kgCarbonValue, index) => {
+  const useMetricUnits = useSelector(userPreferences.selectors.getUseMetricUnits);
+  const getDisplayUnitsValue = calculation.getDisplayUnitsValue;
+  const getDisplayUnits = calculation.getDisplayUnits;
+
+  return (
+    <Text.Secondary center key={index} style={styles.worldExampleItem}>
+      {t(translation)}
+      <FormattedNumber
+        value={Math.round(getDisplayUnitsValue(kgCarbonValue, useMetricUnits))}
+      />{" "}
+      {getDisplayUnits(kgCarbonValue, useMetricUnits, !(kgCarbonValue > 1000 && useMetricUnits))}
```

Comment:
Can be simplified, as we only want the short version so it's more readable : 
```
{getDisplayUnits(kgCarbonValue, useMetricUnit)}
{"CO2eq"}
```
We are also missing the co2eq for some reason

Reply:
Ah okay. I was trying to maintaining the long version for tons to keep the UI the way it was before, but okay yeah I'll make the change so that for all units only the symbol is displayed here. Also oops on the co2eq